### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by realekhor**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```
Comment (ewingjm) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]: I think these defaults should be taken out.

Comment (ewingjm) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]: I think the default Desktop and Smartphone headers should be taken out.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
```
Comment (ewingjm) [inline .github/ISSUE_TEMPLATE/feature_request.md:16]: I've taken this default header out of the other repositories as it doesn't seem that useful

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```
Comment (ewingjm) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]: I'd update the issue template to automatically tag it as a bug for easier querying

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
```
Comment (ewingjm) [inline .github/ISSUE_TEMPLATE/feature_request.md:5]: Would also update this to tag it as an enhancement

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (realekhor) [reply .github/ISSUE_TEMPLATE/bug_report.md]: done

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (realekhor) [reply .github/ISSUE_TEMPLATE/feature_request.md]: done

File: azure-pipelines-pull-request.yml [hunk 5]
```diff
@@ -14,23 +14,40 @@ stages:
       steps:
```
[the remark this answers is not in the document]
Comment (realekhor) [reply azure-pipelines-pull-request.yml:14]: yeah, i thought about splitting it but was trying to get just enough to get this in. but it is easy to do. i will add it.

[the remark this answers is not in the document]
Comment (realekhor) [reply azure-pipelines-pull-request.yml]: updated

**Review by ewingjm — APPROVED** [review]
Just a reminder that you will want to make sure the commit title matches the conventional commits spec that's in the CONTRIBUTING.md when squashing

e..g. `docs: added templates and contributing guidance`

**Pull request merged**